gateway/src/main/java/org/georchestra/gateway/security/oauth2/OAuth2Configuration.java

```diff
@@ -152,11 +153,13 @@ OAuth2UserMapper oAuth2GeorchestraUserUserMapper() {
      * claims.
      *
      * @param nonStandardClaimsConfig Configuration for custom OIDC claims.
+     * @param properties              Configuration for custom OIDC claims.
```

Comment:
javadoc modified, but no "properties" param added to the method ?

Reply:
indeed, no "properties" param added to the method